Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraint.cs

```diff
@@ -361,6 +401,7 @@ protected virtual Vector3 CalculateGoalPosition()
                     if (palmPose.HasValue)
                     {
                         goalPosition = palmPose.Value.Rotation * (localSpaceHit) + palmPose.Value.Position;
+                        goalPosition += (CameraCache.Main.transform.position - goalPosition).normalized * ForwardOffset;
```

Comment:
Do you think it's worth checking if `ForwardOffset` is non-`Vector3.zero` before calculating this normalized direction? I know normalizing can have a bit of a perf cost, but not sure if the Vector3 comparison would be better or worse than that...

Reply:
Done!